commands/keplr.js

```diff
@@ -93,6 +93,88 @@ const keplr = {
     );
     return true;
   },
+  async createWallet(password, walletName, selectedChains) {
+    await module.exports.goToRegistration();
+    await playwright.waitAndClickByText(
+      onboardingElements.createWalletButton,
+      playwright.keplrWindow(),
+    );
+
+    await playwright.waitAndClickByText(
+      onboardingElements.createNewRecoveryPhraseButton,
+      playwright.keplrWindow(),
+    );
+    await playwright.waitAndClickByText(
+      onboardingElements.showMyPhraseButton,
+      playwright.keplrWindow(),
+      true,
+    );
+
+    await playwright.waitAndClickByText(
+      onboardingElements.phraseCount24,
+      playwright.keplrWindow(),
+    );
+    await playwright.waitAndClickByText(
+      onboardingElements.copyToClipboardButton,
+      playwright.keplrWindow(),
+    );
+
+    const passPhraseArray = clipboardy.readSync().split(' ');
+    await playwright.waitAndClickByText(
+      onboardingElements.nextButton,
+      playwright.keplrWindow(),
+    );
+
+    const wordLabels = await playwright.waitForElementsByText(
+      /Word #\d+/,
+      playwright.keplrWindow(),
```

Comment:
`/Word #\d+/` is already quite descriptive. Although, if we want to be more specific about the range of digits, we can use: `/Word #([1-9]|1[0-9]|2[0-4])/`

Reply:
i think this would reduce readibilty quite a bit